docs/modules/security/pages/ldap-authentication.adoc

```diff
@@ -0,0 +1,295 @@
+= LDAP Authentication
+:page-enterprise: true
+
+LDAP servers are one of the most popular identity stores.
```

Comment:
Nit - do we want to include a definition of `LDAP`?

Reply:
Updated in 5642398ccf0acfda535f5f89696769c077e580f2